test/e2e/common/node/pod_level_resources.go

```diff
@@ -0,0 +1,319 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubecm "k8s.io/kubernetes/pkg/kubelet/cm"
+	"k8s.io/kubernetes/test/e2e/feature"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+const (
+	Cgroupv2MemLimit  string = "/sys/fs/cgroup/memory.max"
+	Cgroupv2CPULimit  string = "/sys/fs/cgroup/cpu.max"
+	Cgroupv2CPUWeight string = "/sys/fs/cgroup/cpu.weight"
+	CPUPeriod         string = "100000"
+)
+
+var _ = SIGDescribe("Pod Level Resources", framework.WithSerial(), feature.PodLevelResources, "[NodeAlphaFeature:PodLevelResources]", func() {
+	f := framework.NewDefaultFramework("pod-level-resources-tests")
+	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
+
+	ginkgo.BeforeEach(func(ctx context.Context) {
+		_, err := e2enode.GetRandomReadySchedulableNode(ctx, f.ClientSet)
+		framework.ExpectNoError(err)
+		if framework.NodeOSDistroIs("windows") {
+			e2eskipper.Skipf("not supported")
+		}
+	})
+	podLevelResourcesTests(f)
+})
+
+type ContainerInfo struct {
```

Comment:
Is there a need to export the type? Same for the `ResourceInfo`

Reply:
changing to un-exported type until new tests in common in test/e2e/node are added.